docs-src/concepts/what-is-an-activity-definition.md

```diff
@@ -19,6 +19,18 @@ Activity Definitions are named and referenced in code by their [Activity Type](/
 
 ![Activity Definition](/diagrams/activity-definition.svg)
 
+#### Idempotency
+
+Temporal recommends that Activities be idempotent.
+
+Activities are idempotent if multiple [Activity Task Executions](/concepts/what-is-an-activity-task-execution) do not change the state of the system beyond the first Activity Task Execution.
+
+Activities may be tried more than once even if the Retry Policy says they shouldn't. Because of that, you may need to use idempotency keys for critical side effects.
```

Comment:
```suggestion
An Activity can be tried more than once even if the Retry Policy says it shouldn't.
Because of that behavior, you might need to use idempotency keys for critical side effects.
```
Capitalize "Side Effects" and link to glossary term?